skrifa/src/color/traversal.rs

```diff
@@ -447,6 +447,24 @@ pub(crate) fn traverse_with_callbacks(
     }
 }
 
+pub(crate) fn traverse_v0_range(
+    range: &Range<usize>,
+    instance: &ColrInstance,
+    painter: &mut impl ColorPainter,
+) -> Result<(), PaintError> {
+    for layer_index in range.clone() {
+        let (layer_index, palette_index) = (*instance).v0_layer(layer_index)?;
+        // TODO(drott): Use optimized callback function combining clip, fill and transforms.
```

Comment:
Request: file issues rather than TODO's, it's far more visible.

Reply:
(unless you plan to address basically immediately)

Reply:
TODO linking to newly filed issue #746 now.